doc/development.rst

```diff
@@ -5,7 +5,7 @@ To get a development installation of Flit itself::
 
     git clone https://github.com/takluyver/flit.git
     cd flit
-    python3 -m pip install docutils requests toml
+    python3 -m pip install docutils requests toml tomli
```

Comment:
```suggestion
    python3 -m pip install docutils requests tomli tomli-w
```

Reply:
My install didn't complain about tomli-w being missing

Reply:
Ah I see, I think that's because the `bootstrap_dev.py` installs `flit` dependencies if they aren't installed already. So this line only strictly needs to list dependencies that are used by whatever parts of the codebase `bootstrap_dev.py` runs.

I still think this line may be easier to grasp and maintain if it simply installs all flit dependencies.

Regardless, the `toml` dependency should no longer be needed here.

Reply:
I think there are 2 reasons one of them being explained on bootstrap.rst. It's that `tomli` is itself packaged with flit, so to bootstrap on packaging system like conda-forge you want to be able to maybe call `bootstrap_dev`, before tomli/tomli-w are themselves packaged.

Not sure.

Reply:
`bootstrap_dev.py` is really meant for development, rather than downstream packaging - it does an editable install of both flit_core & flit. It might make sense to add another script to help with bootstrapping `flit_core`.